core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -54,33 +62,86 @@ class DefaultPageBuilder(
         }
     }
 
-    private fun contentForClass(c: Class) = group(c) {
-        header(1) { text(c.name) }
-        c.inherited.takeIf { it.isNotEmpty() }?.let {
-            header(2) { text("SuperInterfaces") }
-            linkTable(it)
+    private fun contentForClasslike(c: Classlike): ContentGroup = group(c) {
+        headerForClasslike(c)
+
+        when (c) {
+            is Class -> partialContentForClass(c)
+            is Enum -> partialContentForEnum(c)
+            else -> TODO("$c should not be present here")
```

Comment:
It's not a `TODO`, it designed that way so it should throw some proper exception

Reply:
Changed, it's an exception now.